website/app/styles/app.scss

```diff
@@ -5,6 +5,7 @@
 
 // Tokens (CSS props)
 @import "tokens";
+@import "prism-dracula";
```

Comment:
not sure which method is better, no strong opinions, but just in case see how it's done here: https://github.com/hashicorp/design-system/blob/main/packages/components/ember-cli-build.js#L25

Reply:
The thinking behind placing/vendoring it in an SCSS file was that this way we'll be in full control of the import order – which could matter when dealing with overrides. This was the `code-block.scss` will always be loaded after the theme.

Reply:
good thinking! didn't realize it. 👍